src/browser/web-ai/chatgpt-response-dom.ts

```
@@ -12,60 +15,396 @@ export const CHATGPT_ASSISTANT_SELECTORS: string[] = [
+    };
+
+    for (const selector of stopSelectors) {
+        let nodes: AnyNode;
+        try { nodes = document.querySelectorAll(selector); } catch { continue; }
+        if (Array.from(nodes).some(isVisible)) return { strength: 'strong', evidence: 'stop-button' };
+    }
+
+    let assistantNodes: AnyNode[] = [];
+    try {
+        const resolver = resolverSource
+            ? (0, eval)(`(${resolverSource})`)
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)